geode-core/src/test/java/org/apache/geode/distributed/internal/InternalClusterConfigurationServiceTest.java

```diff
@@ -87,9 +88,13 @@ public void jndiBindings() {
       jndiBinding.setJndiName("jndiOne");
       jndiBinding.setJdbcDriverClass("com.sun.ABC");
       jndiBinding.setType("SimpleDataSource");
-      jndiBinding.getConfigProperty()
-          .add(new JndiBindingsType.JndiBinding.ConfigProperty("test", "test", "test"));
-      cacheConfig.getJndiBindings().add(jndiBinding);
+
+      ConfigProperty configProperty = new ConfigProperty();
+      configProperty.setName("test");
+      configProperty.setType("test");
+      configProperty.setValue("test");
+      jndiBinding.getConfigProperties().add(configProperty);
+      cacheConfig.getJndiBindings().getJndiBindings().add(jndiBinding);
```

Comment:
should be `cacheConfig.getJndiBindings().add(jndiBinding)

Reply:
Whoops!  Looks like my build alias skips compiling on the test side.  That's something I need to fix!